docs/cn/Web-Interface.md

```diff
@@ -41,7 +41,7 @@ Alluxio master 主页如下所示:
 
 ## 配置页面
 
-查看当前系统的配置信息，点击屏幕上方导航栏的"System Configuration"。
+查看当前系统的配置信息，点击屏幕上方导航栏的"Configuration"。
```

Comment:
why do you want to delete "System" ?
The actual Web UI has "System Configuration" navigation bar.